src/led.h

```diff
@@ -0,0 +1,31 @@
+/* Fidelio
+ *
+ * (c) 2023 Daniele Lacamera <root@danielinux.net>
+ *
+ *
+ * Fidelio is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Fidelio is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ *
+ */
+#ifndef U2F_LED_H
+#define U2F_LED_H
+
+#include "pico/stdlib.h"
+
+#define U2F_LED PICO_DEFAULT_LED_PIN
```

Comment:
I think it's mentioned somewhere pins.h is where users change pin configuration, i.e. button + led, that's why I asked to restore it. Apologies for not clarifying this better.

Perhaps we could think of making this even nicer with `#ifndef U2F_LED`, so that it could be compiled with e.g. `cmake -DU2F_LED=3 -DPRESENCE_BUTTON=4` without touching the code. Maybe `U2F_LED` could even override the default CYW43 led?

If you move the U2F_LED define back to pins.h, then you can move the two function decl. below to pins.h and remove led.h, or keep led.h just for those. Either is OK for me.

Reply:
I'll stick it back in pin.h for now but there's more to think about...

One issue here is that a Pico W has no default LED, you have to go thru the WiFi chip - this is what prompted me moving LED stuff into functions. The Waveshare has no LED at all - it's got a little LCD instead.

If the user requests a -DU2F_LED=n then I guess that should override the defau;ts (e.g. external LED being used)
A futher expansion would be -DU2F_LED=-1 which would signify doesn't use LED at all (as 0 is a valid pin) - this would allow the use of e.g. waveshare LCD

Coming up (tomorrow I hope) I want to also add in the BOOTSEL button as an option - this one also doesn't have a pin...

Longer term (next week?) I'm going to add LCD - hopefully flexible enough for multiple device options (as the ST7789 is everywhere) but should also be overridable